.gitignore

```diff
@@ -1,3 +1,4 @@
 node_modules
 /sinopiautil.js
 lib
+PagePicker/.vscode/settings.json
```

Comment:
Actually, I think VS Code user should commit this file. See the gitignore template [here](https://github.com/github/gitignore/blob/master/Global/VisualStudioCode.gitignore)